src/clevercloud/CleverLemStem.py

```diff
@@ -1,21 +1,36 @@
+
 def CleverLemStem(text):
     """
     A preprocessor to conduct lemmatinzation and stemming on the text.   
     
     Parameters
     ----------
-    text : numpy.array
-        Input an array of text / strings 
+    text : str
+        Input a strng 
```

Comment:
```suggestion
        Input a string
```
Hi, string is misspelt here